components/Press/PressLinks/PressLinks.css

```diff
@@ -0,0 +1,34 @@
+.logos a {
+  color: #249cbc;
+  text-decoration: none;
+}
+
+.logos .articlesGroup > h2 {
+  text-align: center;
+}
+
+.logos .articlesGroup {
+  width: 375px;
+  padding-top: 15px;
+}
+
+@media screen and (max-width: 414px) {
+  .logos .articlesGroup {
+    width: 275px;
+  }
+}
+
+.flexContainer {
+  width: 100%;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-around;
+  padding: 15px 0 0 0;
+}
+
+button.ShowAllButton {
```

Comment:
you can just apply it to the class and not involve `button`. I'd prefer that 😀

Reply:
Some of the css atributes being used is also being used by the button component itself.
when changing the css values of those the buttons class is winning the specificity war.
Adding button was a cheap hack to win the war, any other ideas?

Reply:
the class is nested, right? Just do `parent .showAllButton` or `parent parent .showAllButton`

Reply:
of course.... fixed, thanks.